test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -2193,20 +2193,18 @@ def test_cgroup_cpuset_memory_spread_page(self):
         o = j.attributes[ATTR_o]
         self.tempfile.append(o)
         spread_path = 'cpuset.memory_spread_page'
-        fn = self.get_cgroup_job_dir('cpuset', jid, self.hosts_list[0])
+        fn = self.get_cgroup_job_dir('cpuset', jid, hostn)
         if self.noprefix:
             spread_path = 'memory_spread_page'
         fn = os.path.join(fn, spread_path)
-        result = self.du.cat(hostname=self.hosts_list[0],
-                             filename=fn, sudo=True)
+        self.assertTrue(self.is_file(fn, hostn))
+        result = self.du.cat(hostn, filename=fn, sudo=True)
```

Comment:
Here is another case where the old version used "hostname=", so you should use "hostname=hostn".